### PR TITLE
IGNITE-23385 Move fields added to SnapshotMeta from JRaft to our own message

### PR DESCRIPTION
**Pull request by rpuch**
https://issues.apache.org/jira/browse/IGNITE-23385

### CODE REVIEW

File: modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/network/PartitionReplicationMessageGroup.java [hunk 1]
```diff
@@ -197,6 +198,10 @@ public interface PartitionReplicationMessageGroup {
      * Message type for {@link UpdateMinimumActiveTxBeginTimeReplicaRequest}.
      */
     short UPDATE_MINIMUM_ACTIVE_TX_TIME_REPLICA_REQUEST = 26;
+    /**
```
Comment (sashapolo) [inline modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/network/PartitionReplicationMessageGroup.java:201]:
```suggestion
    
    /**
```

File: modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/network/raft/SnapshotMetaResponse.java [hunk 2]
```diff
@@ -20,11 +20,10 @@
...
 import org.apache.ignite.internal.network.annotations.Transferable;
 import org.apache.ignite.internal.partition.replicator.network.PartitionReplicationMessageGroup;
-import org.apache.ignite.raft.jraft.entity.RaftOutter.SnapshotMeta;
 
 /** Snapshot meta response message. */
 @Transferable(PartitionReplicationMessageGroup.SNAPSHOT_META_RESPONSE)
 public interface SnapshotMetaResponse extends NetworkMessage {
     /** Snapshot meta. */
-    SnapshotMeta meta();
+    PartitionSnapshotMeta meta();
```
Comment (sashapolo) [inline modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/network/raft/SnapshotMetaResponse.java:28]: I think we also need to rename the request and response classes to show that they are only applicable to partitions

Comment (rpuch) [reply modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/network/raft/SnapshotMetaResponse.java:28]: It's in a package related to partitions, so it seems to be already clear from the context. WDYT?

Comment (sashapolo) [reply modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/network/raft/SnapshotMetaResponse.java:28]: Up to you, I won't insist

**Review by sashapolo — APPROVED** [review]

**Pull request merged**